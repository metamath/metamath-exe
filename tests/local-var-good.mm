  $c formula $.
  $c A. $.
  $c setvar $.
  $v ph $.
  wph $f formula ph $.

  ${
    $v x $.
    $( Let ` x ` be an individual variable (temporary declaration). $)
    vx.wal $f setvar x $.
    $( Extend formula definition to include the universal quantifier ('for all').  $)
    wal $a formula A. x ph $.
  $}

