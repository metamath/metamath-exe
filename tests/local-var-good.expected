MM> READ "local-var-good.mm"
Reading source file "local-var-good.mm"... 313 bytes
313 bytes were read into the source buffer.
The source has 10 statements; 1 are $a and 0 are $p.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> 0 10%  20%  30%  40%  50%  60%  70%  80%  90% 100%
..................................................
All proofs in the database were verified.
